packages/earth-map/src/sagas/location/index.tsx

```diff
@@ -93,10 +95,18 @@ function* toLocation({ payload, meta }) {
     yield put(setPlaceSelectedFilter(''));
     yield put(setPlaceSelectedSearch(''));
     yield put(setPlaceData(formattedData));
+    yield put(setLastViewedPlace({
+      id: data.id,
```

Comment:
destructure data

Reply:
That won't code more readable and would pollute the function scope with unnecessary variable declarations.